cla_public/apps/base/extensions.py

```diff
@@ -11,3 +11,13 @@ def asset(filename, min_ext="min"):
         filename = ".".join(amended)
 
     return url_for("static", filename=filename)
+
+
+@base.app_template_global()
+def is_quick_exit_enabled(nodes, session=None):
+    if "diagnosis_previous_choices" in session.checker:
+        quick_exit_categories = ["n43n3", "n18", "n19", "n88", "n86", "n97", "n149"]
+        for category in quick_exit_categories:
+            if category in session.checker["diagnosis_previous_choices"]:
+                return category
```

Comment:
If we're returning the category, we might consider renaming the function as the current name suggests a boolean return to me. "Is it enabled?" "n43n3" is a bit unexpected to me. But see the comment in the other file first, for whether we actually need to return the category (the previous version, in the template, didn't)